src/main/resources/swagger.api/queryTool.yaml

```diff
@@ -133,7 +133,74 @@ paths:
           description: Query has been deleted
         '404':
           description: Query with id not found
-
+  /query/{queryId}/sortedIds:
+    get:
+      operationId: getSortedIds
+      description: Get ids of query results in sorted order
+      tags:
+        - fqlQuery
+      parameters:
+        - $ref: '#/components/parameters/queryId'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/limit'
+      responses:
+        '200':
+          description: 'Details of the submitted query'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/queryResultIds'
+        '400':
+          $ref: '#/components/responses/badRequestResponse'
+        '500':
+          $ref: '#/components/responses/internalServerErrorResponse'
+  /query/contents:
```

Comment:
This may not be the best endpoint, since the method is really just getting the contents for a list of ids (no query required)